src/main/java/com/eprosima/fastcdr/idl/templates/TypesSwigInterface.stg

```diff
@@ -14,6 +14,8 @@
 
 group TypesSwigInterface;
 
+import "eprosima.stg"
+
```

Comment:
I think this change can be reverted as the methods declared in that template file are not being used in this file. Besides, it should probably include `import "com/eprosima/fastdds/idl/templates/eprosima.stg"`.

Reply:
Done in 3d798c2